### ISSUE TITLE
compas2 and compas1 serialization

### ISSUE THREAD

**Issue by petrasvestartas** [compas-dev/compas#1211]
**Describe the bug**
How **compas1 .json** files can be de-serialized in **compas2 alpha**?

For now I do like this:
* I install compas 1 stable version
* load geometry in Rhino through artists
* change the environment where v2 is installed
* then install compas2 alpha and serialize it again
* then load to compas vs code

A bit tedious process. I am wondering if it is possible to have a serialization process where you can specify the version number of compas. 

**To Reproduce**
E.g. repository .json files:
https://github.com/BlockResearchGroup/compas_assembly/tree/main/src/compas_assembly/data/samples

```python
from compas.data import json_load

if __name__ == "__main__":
    meshes = json_load('src/compas_assembly2/data_sets/crossvault.json')
```

**Expected behavior**
Working

**Screenshots**
<img width="913" alt="image" src="https://github.com/compas-dev/compas/assets/18013985/6b831fa5-c3c0-470b-a95d-f2b7ea08b6af">

**Desktop (please complete the following information):**
 - OS: Win
 - Python version 3.11.6
 - Python package manager conda

**Comment by tomvanmele** [comment]
yes, i know. there is a slight change in the data serialisation format between COMPAS 1 and 2. i will add a conversion script...

**Comment by xarthurx** [comment]
> yes, i know. there is a slight change in the data serialisation format between COMPAS 1 and 2. i will add a conversion script...

Is this implemented?
